typescript/lib/config.ts

```diff
@@ -506,6 +525,37 @@ export class AIConfigRuntime implements AIConfig {
     delete this.metadata.models[modelName];
   }
 
+  /**
+   * Sets the model to use for all prompts by default in the AIConfig.
+   * @param modelName The name of the model to default to.
+   */
+  public setDefaultModel(modelName: string | undefined) {
+    if (modelName == null) {
+      delete this.metadata.default_model;
+      return;
+    }
+
+    this.metadata.default_model = modelName;
+  }
+
+  /**
+   * Adds a model name : model parser ID mapping to the AIConfig metadata. This model parser will be used to parse Prompts in the AIConfig that use the given model.
+   * @param modelName The name of the model to set the parser for.
+   * @param modelParserId The ID of the model parser to use for the model. If undefined, the model parser for the model will be removed.
+   */
+  public setModelParser(modelName: string, modelParserId: string | undefined) {
```

Comment:
Should this be synced to the ModelParserRegistry?

`config.set_model_parser("gpt-3.5-turbo", "HuggingFaceTextParser")` would update the the config but user would then have to manually run
`ModelParserRegistry.register_model_parser`

Could also make sense to make ModelParserRegistry a non static class and have it be an attribute of AIConfigRuntime